docs/components/query-frontend.md

```diff
@@ -81,7 +81,7 @@ config:
   max_get_multi_concurrency: 0
   max_item_size: 0
   max_get_multi_batch_size: 0
-  dns_provider_update_interval: 0s
+  dns_provider_update_interval: 10s
```

Comment:
This default, autogenerated config. Would be nice to find some easy to use defaults generated in - for now, keep it 0 here.

Reply:
Ok! Will leave it at 0